spec/support/load_rake_tasks.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.configure do |config|
```

Comment:
Why do we need to do this?

Reply:
To load rake tasks and be able to test rake tasks

Reply:
I'd never seen it done like this before each test, I think we can avoid this and just put the `Rails.application.load_tasks` line in the `spec_helper.rb` like https://github.com/alphagov/content-publisher/blob/f633e044325f7d616e44ae1c6b041f0b9b52e1e0/spec/spec_helper.rb#L25. (`content-publisher` is one of our newer apps and often used as a guide for best practices)

Reply:
I can do that but is far away from best practices. 
If you end up calling `Rails.application.load_tasks` twice without a cleanup, your rake tasks will run twice at invocation in the code. I solved bugs for it recently on a code base so I am quite confident that this is the way to go. Also, this is not done before each test, only the ones with the correct tag will run this

Reply:
Okay, that sounds reasonable 👍